definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-general-referral-nonprod.json

```diff
@@ -25,7 +25,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "GeneralReferralReason",
     "UserRole": "caseworker-divorce-courtadmin-la",
-    "CRUD": "R"
+    "CRUD": "CRUD"
```

Comment:
Why is it that we have CRUD on these ? Would CRU not suffice

Reply:
I'm not sure why, but it didn't work for events triggered more than once if we didn't have `D`.